org.elasticsearch.indices.stats.IndexStatsIT.testFilterCacheStats failed on master

The following can be found in logs:
```
13:26:37 org.elasticsearch.indices.stats.IndexStatsIT > testFilterCacheStats FAILED
13:26:37     java.lang.AssertionError: 
13:26:37     Expected: <0L>
13:26:37          but: was <1L>
13:26:37         at __randomizedtesting.SeedInfo.seed([DAF829FE6C77DB35:BFC7D4BBDC7FDC4C]:0)
13:26:37         at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
13:26:37         at org.junit.Assert.assertThat(Assert.java:956)
13:26:37         at org.junit.Assert.assertThat(Assert.java:923)
13:26:37         at org.elasticsearch.indices.stats.IndexStatsIT.testFilterCacheStats(IndexStatsIT.java:1076)
1
```

```
REPRODUCE WITH: ./gradlew ':server:integTest' --tests "org.elasticsearch.indices.stats.IndexStatsIT.testFilterCacheStats" \
  -Dtests.seed=DAF829FE6C77DB35 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-SZ \
  -Dtests.timezone=Asia/Rangoon \
  -Dcompiler.java=12 \
  -Druntime.java=11
```
```
REPRODUCE WITH: ./gradlew ':server:integTest' --tests "org.elasticsearch.indices.stats.IndexStatsIT.testFilterCacheStats" \
  -Dtests.seed=DAF829FE6C77DB35 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-SZ \
  -Dtests.timezone=Asia/Rangoon \
  -Dcompiler.java=12 \
  -Druntime.java=11
```

Does not reproduce locally.


Build | 20190913105531-AA62B92F
-- | --
Log | https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+periodic/604/console
Build Scans | https://gradle-enterprise.elastic.co/s/pzij6ulgmz4f6https://gradle-enterprise.elastic.co/s/briwqyyrh4l4uhttps://gradle-enterprise.elastic.co/s/2cqozlknuhc4g

--- a/server/src/test/java/org/elasticsearch/indices/stats/IndexStatsIT.java
+++ b/server/src/test/java/org/elasticsearch/indices/stats/IndexStatsIT.java
@@ -62,7 +62,6 @@ import org.elasticsearch.test.ESIntegTestCase;
 import org.elasticsearch.test.ESIntegTestCase.ClusterScope;
 import org.elasticsearch.test.ESIntegTestCase.Scope;
 import org.elasticsearch.test.InternalSettingsPlugin;
-import org.elasticsearch.test.junit.annotations.TestLogging;
 
 import java.io.IOException;
 import java.util.ArrayList;
@@ -974,7 +973,6 @@ public class IndexStatsIT extends ESIntegTestCase {
         assertEquals(total, shardTotal);
     }
 
-    @TestLogging(value = "_root:DEBUG", reason = "https://github.com/elastic/elasticsearch/issues/46701")
     public void testFilterCacheStats() throws Exception {
         Settings settings = Settings.builder().put(indexSettings())
             .put("number_of_replicas", 0)

